README.md

```
@@ -1,9 +1,242 @@
+| Package                                       | Description                             | Path                  |
+| --------------------------------------------- | --------------------------------------- | --------------------- |
+| [@gladiaio/sdk](packages/sdk-js/README.md)    | JavaScript / TypeScript SDK             | `packages/sdk-js`     |
+| [gladiaio-sdk](packages/sdk-python/README.md) | Python SDK                              | `packages/sdk-python` |
+| generator                                     | OpenAPI → SDK type generator (internal) | `packages/generator`  |
 
-You can also take a look at [our code samples](https://github.com/gladiaio/gladia-samples).
+You can also check out [our code samples](https://github.com/gladiaio/gladia-samples).
+
+## Repository architecture
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)